src/quo2/components/dividers/strength_divider/component_spec.cljs

```diff
@@ -0,0 +1,18 @@
+(ns quo2.components.dividers.strength-divider.component-spec
+  (:require [quo2.components.dividers.strength-divider.view :as strength-divider]
+            [test-helpers.component :as h]))
+
+(h/describe "select-profile component"
+  (h/test "render component"
+    (h/render [strength-divider/view {:status :okay}])
+    (-> (h/expect (h/get-by-label-text :strength-divider))
+        (.toBeTruthy)))
+  (h/test "render component with correct status"
+    (h/render [strength-divider/view {:status :strong}])
+    (-> (h/expect (h/get-by-text "Strong"))
```

Comment:
When rendering the component for tests, i18n labels are not loading.

<img width="809" alt="Screenshot 2023-02-25 at 01 31 24" src="https://user-images.githubusercontent.com/19279756/221280061-c108e5f6-6287-4064-b924-33b0205bf2e3.png">

I tried using `(i18n/set-language "en")` & `(i18n-resources/load-language "en")` after referring to [this test file](https://github.com/ajayesivan/status-mobile/blob/d030e211e302acb9f08cbaff7252912d01169a47/src/status_im2/contexts/communities/menus/community_options/component_spec.cljs#L9) but it didn't help. 

Couldn't figure out how to make i18n labels work in test renders. @J-Son89 could you please help with this?

Reply:
Strange, yeah for sure I will take a look!

Reply:
so I figured out what the issue with that test is
as far as I can tell, no other test is actually using a translation yet
so the test suite was not configured correctly just yet
so the solution is this
in "test/jest/jestSetup.js"
you can add the codeblock
```
jest.mock("i18n-js", () => ({
    ...jest.requireActual("i18n-js"),
    t: (label) => `tx:${label}`
}))
 ```
this will mock the function
but just return the keyword and not the translation
so then in your test it would be:
```
(h/get-by-text "tx:strength-divider-status-strong")
```

Reply:
@briansztamfater what do you think about this? I believe we discussed mocking translations before (in the other unit tests though)